aspnetcore/fundamentals/http-requests.md

```diff
@@ -588,7 +588,7 @@ Keeping a single `HttpClient` instance alive for a long duration is a common pat
 Using `IHttpClientFactory` in a DI-enabled app avoids:
 
 * Resource exhaustion problems by pooling `HttpMessageHandler` instances.
-* Stale-DNS problems by cycling `HttpMessageHandler` instances at regular instances.
+* Stale DNS problems by cycling `HttpMessageHandler` instances at regular intervals.
```

Comment:
@guardrex @scottaddie If we revert to the original 'stale-DNS problems' here (and I do agree with @guardrex 's logic), then should the line above also be 'resource-exhaustion problems' (ie add hyphen?)  (and also in the two similar places)

Reply:
:smile: I'm kind'a sorry that I brought it up. 🤦‍♂

Yeah ... one could make that case. However, we're in the weeds here on *nit*-ish things. There's no confusion in either case. Let's leave this alone and move on. 🏃

Reply:
I still think we're better off without the hyphen. Let's move on.